src/languages/_english.json

```diff
@@ -1191,6 +1191,27 @@
         },
         "header": "Swaps on Rainbow",
         "subheader": "REINTRODUCING"
+      },
+      "notifications_launch": {
+        "primary_button": {
+          "permissions_enabled": "􀜊 Configure",
```

Comment:
i think most of the time the icons are outside of the i18n text(in our current uses) don't care either way just should prob standardize